firebase-firestore/src/main/java/com/google/firebase/firestore/core/Bound.java

```diff
@@ -78,6 +76,17 @@ public String canonicalString() {
 
   /** Returns true if a document sorts before a bound using the provided sort order. */
   public boolean sortsBeforeDocument(List<OrderBy> orderBy, Document document) {
+    int comparison = compareToDocument(orderBy, document);
+    return inclusive ? comparison <= 0 : comparison < 0;
+  }
+
+  /** Returns true if a document sorts after a bound using the provided sort order. */
```

Comment:
readability request: can you include this [comment](https://github.com/firebase/firebase-android-sdk/pull/2979#discussion_r709612724) somewhere in the class to save future us from going through this again?

Reply:
Done (added an abbreviated version to `isInclusive`).